app/routes/contact.js

```diff
@@ -13,8 +13,13 @@ export default Route.extend({
             newContact.save().then(() => newContact.set('responseMessage', true));
         },
 
-        willTransaction() {
-            this.controller.get('model').rollbackAttributes();
+        willTransition(transition) {
```

Comment:
great! I was hoping your tests could point to that bug. It is always better to find your own bugs ;)

If I remember well there is at least another place you wrote the name of the hook wrongly

Reply:
I corrected before. This one I founded out now!!! Thanks